src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag

```diff
@@ -122,6 +122,7 @@
                                 </label>
                             </h5>
                             <div class="col-sm-10">
+                                <div id="richToolbarContainer"></div>
```

Comment:
Use `this-kind-of-case` for div IDs (although there are myriad of violations across the codebase; we're currently working to fix that), so this one should be `richtext-toolbar-container`.